flex-linux-setup/flex_linux_setup/flex_setup.py

```
@@ -850,6 +859,23 @@ def generate_gluu_passwurd_api_keystore(self):
+        Config.templateRenderingDict['agama_pw_deployment_start_date'] = self.get_ldap_time()
+        Config.templateRenderingDict['agama_pw_assets_base64'] = config_api_installer.generate_base64_file(self.agama_pw_fn, 1)
+        config_api_installer.renderTemplateInOut(
+            self.agama_pw_deployment_ldif_fn,
+            self.templates_dir,
+            self.source_dir
+        )
+
+        agama_pw_deployment_ldif_rendered_fn = os.path.join(self.source_dir, os.path.basename(self.agama_pw_deployment_ldif_fn))
+        self.dbUtils.import_ldif([agama_pw_deployment_ldif_rendered_fn])
+        print("Enabling Agama Script")
+        installer_obj.dbUtils.enable_script('BADA-BADA')
```

Ruff (0.16.1)

[error] Undefined name `installer_obj`

(F821)